mne/viz/tests/test_raw.py

```diff
@@ -421,6 +423,12 @@ def test_plot_raw_psd():
     raw.plot_psd(picks=picks, average=False)
     raw.plot_psd(picks=picks, average=True)
     plt.close('all')
+    old_log_level = set_log_level(logging.CRITICAL)
+    raw.set_channel_types({'MEG 0113': 'hbo', 'MEG 0112': 'hbr',
+                           'MEG 0122': 'fnirs_raw', 'MEG 0123': 'fnirs_od'})
+    set_log_level(old_log_level)
+    fig = raw.plot_psd()
+    assert len(fig.axes) == 10
```

Comment:
Yes if this had been 9 in master but with this PR it's 10 (or would have been 8 and not it's 10 or whatever) this is what I had in mind

Reply:
Great. It increased by 4 because I added 4 types. Makes sense. Thanks